docs/reference/mapping/types/numeric.asciidoc

```diff
@@ -92,3 +93,53 @@ The following parameters are accepted by numeric types:
     (default).
 
 
+==== fixed Type
```

Comment:
Is `fixed` going to be the right name here? unlike `float`, `double`, `integer` and `long` it doesn't immediately suggest a numeric field. Might users be confused by this name and this `fixed` refers to something other than a fixed-point numeric field? (maybe they might think it is some kind of field type that is "fixed" in some other way)